test/test_fakeredis.py

```diff
@@ -2112,6 +2112,37 @@ def test_zadd_incr(r, ch):
     assert zadd(r, 'foo', {'three': 1.0}, incr=True, xx=True, ch=ch) == 4.0
 
 
+def test_zpopmin_empty(r):
+    assert r.zpopmin('foo') == []
```

Comment:
Can you also add a test that doesn't specify a count but for which the set is not empty, to test the code that defaults it to 1.